### PR TITLE
Slideshows

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

**Comment by quarterto** [comment]
Looking good!

screenshot 2016-03-09 15 57 12

**Comment by matthew-andrews** [comment]
why no o-galleries?

**Comment by matthew-andrews** [comment]
[ ahh, probably becasue amp ]

**Comment by georgecrawford** [comment]
I didn't even consider it as I can't imagine the markup is compatible.

**Comment by georgecrawford** [comment]
I've decided to go with titles outside of the box, rather than superimposed, so we get:
screenshot 2016-03-09 16 34 21
...rather than Next's clash:
screenshot 2016-03-09 16 35 49

**Comment by georgecrawford** [comment]
over to you, @quarterto

File: server/lib/transform-slideshows.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	items.reduce((previous, current) => previous + current[dimension], 0) / items.length;
+
+module.exports = function run(article, options) {
+	const $ = cheerio.load(article.htmlBody, {decodeEntities: false});
+
+	return getTemplate(options.production)
+	.then(template => {
+		$('ft-slideshow').map((i, el) => {
+			const uuid = el.attribs['data-uuid'];
+			const slideshow = article.slideshows[uuid];
```
Comment (quarterto) [inline server/lib/transform-slideshows.js]: If fetch-slideshows fails, `article.slideshows` is `undefined` here. Either we should guard here, or set `article.slideshows` earlier in fetch-slideshows.

Comment (georgecrawford) [inline server/lib/transform-slideshows.js]: Thanks - good spot

**Pull request merged**